pkg/node/store/store.go

```diff
@@ -59,14 +66,14 @@ type NodeRegistrar struct {
 }
 
 // RegisterNode registers the local node in the cluster
-func (nr *NodeRegistrar) RegisterNode(n *node.Node) error {
+func (nr *NodeRegistrar) RegisterNode(n *node.Node, manager *nodemanager.Manager) error {
```

Comment:
We are slowly moving again to all packages depending on all other packages. Consider writing an interface for `nodemanager.Manager` instead so that `NodeRegistrar` does not import `nodemanager` package.

Reply:
Very good point. Done